ls1intum/Artemis
# `Development`: Add student code of conduct and apso as a static assets

### Checklist
#### General

### Motivation and Context
The student code of conduct linked in the template of plagiarism case post and linked in the post created by continuous plagiarism control was no longer available. 

Original thread on Slack: https://ls1tum.slack.com/archives/G6TCVL6HL/p1696873741965929.

### Description
The student code of conduct is now a static asset of Artemis. 

### Steps for Testing
Prerequisites:
1. Log in to Artemis.
2. Go to `TEST_SERVER_URL/content/documents/student-code-of-conduct.pdf` and check if the PDF loads.

### Review Progress
#### Code Review
#### Manual Tests
